docs/FAQ/how-to-deploy-bold-bi-and-bold-reports-in-an-ack-cluster.md

```diff
@@ -0,0 +1,337 @@
+
+## Steps for Bold BI and Bold Reports auto deployment using kubectl.
+
+1. Download the deployment file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/239310-Bold-BI-Alibaba-Cloud-Kubernetes-(ACK)-Doc/deploy/deploy/common_ack_deploy.yaml) to deploy Bold BI and Bold Reports on AKS.
+
+2. Navigate to the folder where the deployment files were downloaded in **Step 1**.
+
+3. Follow the documentation provided at [this link](https://www.alibabacloud.com/help/en/nas/latest/create-a-nas-file-system) to create a `file system` in Alibaba Cloud.
+
+4. Once the file system is created, locate and click on the cube icon. This will allow you to view the mount target hostname. Take note of the mount target hostname, as you will need it for mounting the file system with our application.
+
+   ![NAS-Mount-Target](images/Nas-mount-target.png)
+
+5. Open the **common_ack_deploy.yaml** file and replace the <host_name_of_mount_target> placeholder with the actual mount target hostname noted in the previous step.
+
+    ![PV Claim](images/ack_pv_claim.png)
+
+6. If you have a DNS to map with the application, proceed to the next steps, otherwise, skip to **Step 10**. 
+
+7. In common_ack_deploy.yaml file and locate line 2335. Uncomment the host value and replace example.com with your DNS hostname.
+
+      ![DNS](images/ack_ingress_http_yaml.png)
+  
+8. If you have an SSL certificate for your DNS and need to configure the site with it, follow these steps; otherwise, skip to **Step 10**.
+
+9. In the ingress spec of the common_ack_deploy.yaml file, uncomment the tls section. Replace example.com with your DNS hostname and save the file.
+
+      ![ingress DNS](images/ack_ingress_tls_yaml.png)
+
+10. Run the following command to retrieve the External IP address:
+
+      ```sh
+      kubectl get svc -n kube-system
+      ```
+      Repeat the above command until you find the IP address listed in the EXTERNAL-IP column, as shown in the following image:
+
+      ![Ingress Address](images/ack_ingress_address.png)
+
+11. Note the EXTERNAL-IP address and map it to your DNS if you have added it in the common_ack_deploy.yaml file. If you do not have a DNS, use the EXTERNAL-IP address to access the application.
+
+12. Replace `<application_base_url>` with your DNS or EXTERNAL-IP address.
+
+    Examples: `http://example.com`, `https://example.com`, `http://<external_ip_address>`
+    
+    ![App_Base_Url](images/ack_app_base_url.png) 
+    
+13. Read the optional client library license agreement from the following link.
+
+    [Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+    
+14. By default, all client libraries for Bold BI and Bold Reports will be installed in Kubernetes. However, you can overwrite them by specifying the required libraries as a comma-separated list in the environment variable noted from the above link.
```

Comment:
14. By default, all client libraries for Bold BI and Bold Reports will be installed in Kubernetes. However, you can overwrite them by specifying the required libraries as a comma-separated list in the environment variable noted from the above link.